public/utils/constants.ts

```diff
@@ -53,7 +53,8 @@ export const ANOMALY_RESULT_INDEX = '.opendistro-anomaly-results';
 
 export const BASE_DOCS_LINK = 'https://opensearch.org/docs/monitoring-plugins';
 
-export const AD_DOCS_LINK = 'https://opensearch.org/docs/latest/observing-your-data/ad/index/';
+export const AD_DOCS_LINK =
```

Comment:
you might want to rebase the branch, this constant has already been added - https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/featureAnywhere/public/utils/constants.ts#L56

Reply:
I think this is just running prettier on the line